### DISCUSSION TITLE
Modifier priority

### DISCUSSION THREAD

**Discussion by JamesHemery** [nativewind/nativewind#532] [Q&A]
Hi @marklawlor,

It seems that modifiers do not have priority over default classes.

For example : 
```jsx
<Text className="text-slate-900 dark:text-slate-50 text-blue-500">Hello world</Text>
```

The text will be always blue-500.

Any priority management planned? 

Thanks,

**Comment by ecosse3** [comment]
@JamesHemery That's expected in TailwindCSS. Just use https://www.npmjs.com/package/tailwind-merge.

**Reply by JamesHemery** [reply]
No, modifier it should take precedence because it's more precise. Tailwind-merge exists to correct problems when two classes have the same level of precision with tailwind because the CSS order will win over the className order.

Example: https://play.tailwindcss.com/dI1ulJOJvx here the text will be red in dark mode, this is the good behavior. Same code on native will result with blue text in dark mode.

Tailwind-merge is not useful for native styles, as the style is inlined.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
